examples/axis-label-offset/index.js

```diff
@@ -0,0 +1,65 @@
+const domain = [
+    'Carrots',
+    'Bananas',
+    'Sausages',
+    'Pickles',
+    'Aubergines',
+    'Artichokes',
+    'Spinach',
+    'Cucumber',
+    'Beef',
+    'Fish',
+    'Potato',
+    'Avacado',
+    'Cheese',
+    'Marzipan',
+    'Peas',
+    'Popcorn'
+];
+
+const scale = d3.scaleBand().domain(domain);
+
+const draw = () => {
+    const renderAxis = (axisFactory, selector, transform) => {
+        const hostElement = d3.select(selector).node();
+        const groupRect = hostElement.getBoundingClientRect();
+
+        scale.range([
+            0,
+            hostElement.getAttribute('data-orient') === 'vertical'
+                ? groupRect.height
+                : groupRect.width
+        ]);
+
+        const axis = axisFactory(scale);
+
+        const join = fc.dataJoin('g', 'axis');
+        join(d3.select(selector), [{}])
+            .attr('transform', transform)
+            .call(axis);
+    };
+
+    renderAxis(
+        scale => fc.axisLabelOffset(fc.axisBottom(scale)),
+        '#bottom svg',
+        'translate(0, 0)'
+    );
+    renderAxis(
+        scale => fc.axisLabelOffset(fc.axisTop(scale)),
+        '#top svg',
+        'translate(0, 80)'
```

Comment:
These transforms should be handled like they are in cartesian - https://github.com/d3fc/d3fc/blob/master/packages/d3fc-chart/src/cartesian.js#L126